var/spack/repos/builtin/packages/py-nbconvert/package.py

```diff
@@ -75,13 +76,66 @@ class PyNbconvert(PythonPackage):
     depends_on("py-testpath", when="@5:6.4", type=("build", "run"))
     depends_on("py-lxml", when="@6.5.1:7.0", type=("build", "run"))
 
+    resource(
+        name="index.css",
+        url="https://unpkg.com/@jupyterlab/nbconvert-css@4.0.2/style/index.css",
+        sha256="917ff47850a7cc08fd0658026fda7672a85220aaab258e8849e891b37426f947",
+        placement="resource_index.css",
+        when="@7.4.1:",
+        expand=False,
+    )
+    resource(
+        name="theme-light.css",
+        url="https://unpkg.com/@jupyterlab/theme-light-extension@4.0.2/style/variables.css",
+        sha256="11bf3558fd3ed353a4c1401ac0c1730d01df073f6436d357c5bbf02a03bd6962",
+        placement="resource_theme-light.css",
+        when="@7.4.1:",
+        expand=False,
+    )
+    resource(
+        name="theme-dark.css",
+        url="https://unpkg.com/@jupyterlab/theme-dark-extension@4.0.2/style/variables.css",
+        sha256="795f2d5069737cbeb5cba01e6b5c7cadbde227c909e43004c5a60f58d5160aec",
+        placement="resource_theme-dark.css",
+        when="@7.4.1:",
+        expand=False,
+    )
+    resource(
+        name="style.css",
+        url="https://cdn.jupyter.org/notebook/5.4.0/style/style.min.css",
+        sha256="5865a609f4437b0464bc121cd567b619074e540a0515a3b82f222f764eb51e01",
+        placement="resource_style.css",
+        when="@7.4.1:",
+        expand=False,
+    )
+
+    @run_before("install")
+    @when("@7.4.1:")
+    def install_css(self):
+        css = {
+            # target filename: [subdir, source filename]
+            "index.css": ["lab", "index.css"],
+            "theme-light.css": ["lab", "variables.css"],
+            "theme-dark.css": ["lab", "variables.css"],
+            "style.css": ["classic", "style.min.css"],
+        }
+        for f in css.keys():
```

Comment:
Could do:
```suggestion
        for target, (subdir, source) in css.items():
```
if you want to make it easier to read